src/shared/credentials/credentialsProfileMru.ts

```diff
@@ -15,19 +14,14 @@ export class CredentialsProfileMru {
     public static readonly MAX_CREDENTIAL_MRU_SIZE = 5
 
     private static readonly configurationSettingName: string = 'recentCredentials'
-    private readonly _configuration: SettingsConfiguration
 
-    public constructor(configuration: SettingsConfiguration) {
-        this._configuration = configuration
-    }
+    public constructor(private readonly _context: vscode.ExtensionContext) {}
```

Comment:
Nit: Suggest to use `Pick<vscode.ExtensionContext, 'globalState'>` instead, for easier testing.

Reply:
Think I might pass on this one since we already have the testing utils written (in FakeExtensionContext)